content/influxdb/v2.0/reference/flux/stdlib/experimental/histogramquantile.md

```diff
@@ -0,0 +1,80 @@
+---
+title: experimental.histogramQuantile() function
+description: >
+ The `experimental.histogramQuantile()` function approximates a quantile given a histogram
+ that approximates the cumulative distribution of the dataset.
+menu:
+  influxdb_2_0_ref:
+    name: experimental.histogramQuantile
+    parent: Experimental
+weight: 302
+related:
+  - /influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/histogramquantile/
+---
+
+The `experimental.histogramQuantile()` function approximates a quantile given a histogram that
+approximates the cumulative distribution of the dataset.
+Each input table represents a single histogram.
+The histogram tables must have two columns – a count column (`_value`) and an upper bound column (`le`).
+
+Input tables must include `le` and `_value` columns neither can be part of the group key.
```

Comment:
```suggestion
Each input table represents a single histogram. Input tables must have two columns—a count column (_value) and an upper bound column (le), and neither column can be part of the group key. 

```

Reply:
Is there a reason why we're calling it the count column? The `values` column counts what...

Reply:
Yes, it's how histogram data is structured in Flux. The value column is actually a count of points that fall in a specific bin or range.